common/lib/dependabot/pull_request_creator/github.rb

```diff
@@ -132,6 +139,13 @@ def require_up_to_date_base?
       # rubocop:disable Metrics/PerceivedComplexity
       sig { params(name: String).returns(T::Boolean) }
       def branch_exists?(name)
+        if trace_log?
```

Comment:
this log doesn't seem particularly helpful since we can see if it's a duplicate from the job log in #create, and the sentry error that it raises

Reply:
If we want to know what the branch name is we should maybe change this to a debug statement so it doesn't get printed out with every job log when we remove the ff

Reply:
@Nishnha , i have changed to debug now.